packages/@lwc/integration-karma/test/accessibility/synthetic-cross-root-aria/index.spec.js

```diff
@@ -51,23 +60,54 @@ if (!process.env.NATIVE_SHADOW && !process.env.COMPAT) {
                     it('setting aria-labelledby', () => {
                         expect(() => {
                             elm.linkUsingAriaLabelledBy();
-                        }).toLogWarningDev(expectedMessage);
-                        expect(dispatcher).toHaveBeenCalledWith(
-                            ReportingEventId.CrossRootAriaInSyntheticShadow,
-                            'x-aria-source',
-                            jasmine.any(Number)
-                        );
+                        }).toLogWarningDev([
```

Comment:
nit: might help readability to consolidate this section of ~20 lines checking the expected logged warning and dispatcher calls based on `usePropertyAccess`; on skim it looks like the section is the exact same for 4 of the use cases here.

I can also understand the argument for keeping the checks explicitly under the test case to minimize scrolling around the file.

Reply:
That makes sense to me. Added a commit to DRY the code a bit: 0518409b1